docs/setup/upgrade/upgrade-migrations.asciidoc

```diff
@@ -1,11 +1,13 @@
 [[upgrade-migrations]]
 === Upgrade migrations
 
-Every time {kib} is upgraded it checks to see if all saved objects, such as dashboards, visualizations, and index patterns, are compatible with the new version. If any saved objects need to be updated, then the automatic saved object migration process is kicked off.
+Every time {kib} is upgraded it will perform an upgrade migration to ensure that all saved objects are compatible with the new version.
```

Comment:
Let's add a link to docs explaining what saved objects are. Inlined short explanation would work too.